src/package-fetcher.js

```diff
@@ -87,6 +92,20 @@ export async function fetchOneRemote(
   }
 }
 
+export async function unlinkIgnoredFiles(dest: string): Promise<void> {
```

Comment:
Thank you for working on this! I'm not entirely sure if this is the right place to apply this filter. Could we instead apply this while we unpack so that ignored files are never even written? If we scope it to git dependencies only it shouldn't have an impact on install time.

Reply:
Your welcome! Okay, I will work on it.

Reply:
Thanks for this, how'd you get on? :)

Reply:
hey! sorry, i have been super busy, dealing with the conditions that bring with Covid, but I started to working on again